src/modules/generators/fractals/multifractal.rs

```diff
@@ -0,0 +1,171 @@
+// Copyright 2016 The Noise-rs Developers.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use num_traits::Float;
+use math;
+use math::{Point2, Point3, Point4};
+use NoiseModule;
+use modules::Perlin;
+use super::util::*;
+
+/// Default noise seed for the Multifractal noise module.
+pub const DEFAULT_MULTIFRACTAL_SEED: usize = 0;
+/// Default number of octaves for the Multifractal noise module.
+pub const DEFAULT_MULTIFRACTAL_OCTAVE_COUNT: usize = 6;
+/// Default H for the Multifractal noise module.
+pub const DEFAULT_MULTIFRACTAL_H: f32 = 0.25;
+/// Default lacunarity for the Multifractal noise module.
+pub const DEFAULT_MULTIFRACTAL_LACUNARITY: f32 = 1.91803398875;
+/// Default zero offset for the Multifractal noise module.
+pub const DEFAULT_MULTIFRACTAL_OFFSET: f32 = 0.7;
+/// Maximum number of octaves for the Multifractal noise module.
+pub const MULTIFRACTAL_MAX_OCTAVES: usize = 32;
+
+/// Noise module that outputs Multifractal Perlin noise.
+///
+/// Multifractal Perlin noise is the product of several coherent gradient-noise
+/// functions of ever-increasing frequencies and ever-decreasing amplitudes.
+///
+/// An important property of Multifractal noise is that a small change in the
+/// input value will produce a small change in the output value, while a large
+/// change in the input value will produce a large change in the output value.
+#[derive(Clone, Debug)]
+pub struct Multifractal<T> {
+    /// Seed.
+    pub seed: usize,
+
+    /// Total number of frequency octaves that generate the multifractal noise.
+    pub octaves: usize,
+
+    /// Defines the highest fractal dimension. Fractal Dimension = D - H.
+    pub h: T,
```

Comment:
I reckon that if there are Wikipedia references, we should link to them in the docs so that people can read more. Eg:

```
/// doc comment...
///
/// # References
///
/// - [Hurst Exponent - Wikipedia](https://en.wikipedia.org/wiki/Hurst_exponent)